<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Arqiver::mainWin</class>
 <widget class="QMainWindow" name="mainWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Arqiver</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Archive:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Arqiver::Label" name="label_archive">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Arqiver::TreeWidget" name="tree_contents">
      <property name="dragDropMode">
       <enum>QAbstractItemView::DragOnly</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="iconSize">
       <size>
        <width>22</width>
        <height>22</height>
       </size>
      </property>
      <attribute name="headerStretchLastSection">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item alignment="Qt::AlignLeft">
       <widget class="QLabel" name="label_progress_icon">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="textVisible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Arqiver::Label" name="label_progress">
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>403</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionAdd_File"/>
    <addaction name="actionAdd_Dirs"/>
    <addaction name="actionRemove_File"/>
    <addaction name="separator"/>
    <addaction name="actionExtract_All"/>
    <addaction name="actionExtract_Sel"/>
    <addaction name="separator"/>
    <addaction name="action_Password"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New"/>
   <addaction name="action_Open"/>
   <addaction name="actionAdd_File"/>
   <addaction name="actionAdd_Dirs"/>
   <addaction name="actionRemove_File"/>
   <addaction name="separator"/>
   <addaction name="actionExtract_All"/>
   <addaction name="actionExtract_Sel"/>
  </widget>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open Archive</string>
   </property>
   <property name="statusTip">
    <string>Open archive</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_New">
   <property name="text">
    <string>&amp;New Archive</string>
   </property>
   <property name="statusTip">
    <string>New archive</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAdd_File">
   <property name="text">
    <string>Add File(s)</string>
   </property>
   <property name="statusTip">
    <string>Add files to archive</string>
   </property>
  </action>
  <action name="actionRemove_File">
   <property name="text">
    <string>Remove File(s)</string>
   </property>
   <property name="statusTip">
    <string>Remove selection from archive</string>
   </property>
  </action>
  <action name="actionExtract_All">
   <property name="text">
    <string>Extract All</string>
   </property>
   <property name="statusTip">
    <string>Extract archive into a directory</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionAdd_Dirs">
   <property name="text">
    <string>Add Directory</string>
   </property>
   <property name="statusTip">
    <string>Add directory to archive</string>
   </property>
  </action>
  <action name="actionExtract_Sel">
   <property name="text">
    <string>Extract Selection</string>
   </property>
   <property name="statusTip">
    <string>Extract Selected Items</string>
   </property>
  </action>
  <action name="action_Password">
   <property name="text">
    <string>Set Password</string>
   </property>
  </action>
 </widget>
 <resources/>
 <customwidgets>
  <customwidget>
   <class>Arqiver::Label</class>
   <extends>QLabel</extends>
   <header>label.h</header>
  </customwidget>
  <customwidget>
   <class>Arqiver::TreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>treeWidget.h</header>
  </customwidget>
 </customwidgets>
 <connections/>
</ui>
